cmd/object-api-multipart_test.go

```diff
@@ -1909,7 +1909,9 @@ func testObjectCompleteMultipartUpload(obj ObjectLayer, instanceType string, t T
 		}
 		// Failed as expected, but does it fail for the expected reason.
 		if actualErr != nil && !testCase.shouldPass {
-			if !strings.Contains(actualErr.Error(), testCase.expectedErr.Error()) {
+			_, actualOk := actualErr.(InvalidPart)
```

Comment:
what if `testCase.expectedErr` is `nil`?

Also, what if in both lines 1912 and 1913, errors are not of type `InvalidPart`, but they actually do not match?

Reply:
fixed